src/pages/apply/Apply.jsx

```diff
@@ -0,0 +1,41 @@
+import { FAQTSWebsite, FAQProjectEdu, FAQCFSWebsite, FAQs } from "../../components/faqList/FAQList";
+import ApplyBubble1 from "./bubbles/ApplyBubble1";
+import ApplyBubble2 from "./bubbles/ApplyBubble2";
+import './style.css';
+
+function Apply() {
+  return (
+    <div className="applyContainer">
+      <div className="titleText">
+        <h1>Join us</h1>
+        <p>Make a difference in the world with us!<br/>
+          Found something you can see yourself working on in the list below? Or just considering an unsolicited
+          application?<br/>
+          Great! Just fill in this form and we will get back to you as soon as possible.</p>
+      </div>
+      <div className="positionsText">
+        <h2>Open positions</h2>
+      </div>
```

Comment:
What's the div for? A div is mostly not really usefull when just having one child element, so rather just have the text and then work with margins (outer distance) applied to the text instead of padding (inner distance) applied to the div. ESLint probably also complains about divs with just one child.